microsoft-edge/webview2/release-notes/index.md

```diff
@@ -16,18 +16,135 @@ These Release Notes provide information about new features and bug fixes that ar
 the template to copy for incoming sections is at end of this file
 
 this webpage covers the most recent ~5 months; periodically move oldest h2 sections from bottom of present file to archive.md.  eg covers: 
+Apr 2025
 Mar 2025
 Feb 2025
 Jan 2025
 [Dec 2024]
 Nov 2024
-Oct 2024
 
 moved to archive:
-Sep 2024
+Oct 2024
 -->
 
 
+<!-- Apr 2025 Release SDK -->
+<!-- ====================================================================== -->
+## 1.0.3179.45
+
+Release Date: April 07, 2025
+
+[NuGet package for WebView2 SDK 1.0.3179.45](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3179.45)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 135.0.3179.45 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+No additional APIs have been promoted to Stable and added in this Release SDK.
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Enabled ambient authentication through browser flags.
+* Fixed the \<select\> HTML tag<!-- todo: HTML `Select` component? --> to make it selectable, in WPF apps.
+* Fixed navigation of `edge://crashes` within a WebView2 control.
+* Fixed potential crash and UI issues when invoking the Windows Credentials UI from a WebView2 instance.<!-- https://www.bing.com/search?q=Windows+Credential+UI -->
```

Comment:
check spelling "Windows Credentials UI" - is that what we're referring to?
https://www.bing.com/search?q=Windows+Credential+UI